content/SCALE/SCALETutorials/Storage/ManagePoolsScale.md

```diff
@@ -54,32 +56,148 @@ Select **Confirm Export/Disconnect**
+
+By default, TrueNAS automatically checks every pool on a recurring scrub schedule.
+
+The **ZFS Health** widget displays the state of the last scrub or disks in the pool.
+To view scheduled scrub tasks, click **View all Scrub Tasks** on the **ZFS Health** widget.
+
+## Managing Pool Disks
+
+The **Storage Dashboard** screen **Disks** button and the **Manage Disks** button on the **Disk Health** widget both open the **Disks** screen.
+
+**Manage Devices** on the **Topology** widget opens the ***Poolname* Devices** screen.
+To manage disks in a pool, click on the VDEV to expand it and show the disks in that VDEV.
+Click on a disk to see the devices widgets for that disk.
+You can take a disk offline, detach it, replace it, manage the SED encryption password, and perform other disk management tasks from this screen.
+
+See [Replacing Disks]({{< relref "ReplacingDisks.md" >}}) for more information on the **Offline**, **Replace** and **Online** options.
+
+## Expanding a Pool
+
+You can increase the size of an existing pool by adding one or more drives to an existing RAIDZ VDEV, adding a new VDEV of the same type, or by replacing all existing disks in the VDEV with larger disks.
+
+By default, a VDEV limits all disks to the usable capacity of the smallest attached device.
+TrueNAS SCALE automatically expands the usable capacity of the pool to fit all available space once the smallest attached disk is replaced (see [Replacing Disks to Expand a Pool](#replacing-disks-to-expand-a-pool) below).
+If a pool is not automatically expanded, for example when resizing virtual disks in a hypervisor apart from TrueNAS, click **Expand** on the **Storage Dashboard** to manually increase the pool size to match all available disk space.
+
+### Extending a RAIDZ VDEV
+
+Additional disks can be added one at a time to a RAIDZ VDEV, expanding its capacity incrementally.
+This can be useful for small pools (typically with only one RAID-Z VDEV), where there isn't sufficient hardware capacity to add a mirrored VDEV by doubling the number of disks.
+
+{{< expand "Overview and Considerations" "v" >}}
+TrueNAS SCALE 24.10 (Electric Eel) introduces RAIDZ extension to allow incremental expansion of an existing RAIDZ VDEV using one more disks.
+RAIDZ extension allows resource- or hardware-limited homelab and small enterprise users to expand storage capacity with lower upfront costs compared to traditional ZFS expansion methods.
+
+To expand a RAIDZ array, TrueNAS reads data from the current disks and rewrites it onto the new configuration, including any additional disks.
+
+Data redundancy is maintained.
+If a disk fails mid-expansion, the process pauses until the RAIDZ virtual device (vdev) is healthy again, typically by replacing the failed disk and waiting for the system to rebuild.
```

Comment:
Lets first say that the pool must be healthy to start expansion.

Reply:
```suggestion
Make sure the pool is healthy before beginning the expansion process.
If a disk fails mid-expansion, the process pauses until the RAIDZ virtual device (vdev) is healthy again, typically by replacing the failed disk and waiting for the system to rebuild.
```